submissions/akulahere/css/main.css

```diff
@@ -0,0 +1,155 @@
+body {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    background: #182f35 url(../img/background.jpg) no-repeat fixed center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+header {
+    background: rgba(255,69,0, 0.4);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.step-button {
+    opacity: 1;
+    margin: 10px;
+    text-decoration: none;
+    display: inline-block;
+    padding: 8px 16px;
+    background-color: rgba(76,175,80, 1);
+    color: white;
+    border-radius: 50%;
+}
+
+.step-button:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+footer {
+    background-color: cornflowerblue;
+    display: flex;
+    justify-content: center;
+    opacity: 0.7;
+}
+
+.main-container {
+    height: 70%;
+    max-width: 1640px;
+    align-self: center;
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    grid-template-rows: minmax(70vh, max-content);
+    grid-column-gap: 50px;}
+
+.info-container {
+    min-height: 70%;
+    display: flex;
+    flex-direction: column;
+    background-color: black;
+    background-image: url(../img/movie-logo3.jpg);
+    background-size: 100% 100%;
```

Comment:
Fix image size on the small height of the parent block.

![image](https://user-images.githubusercontent.com/16196199/107271897-8e792300-6a55-11eb-8803-56e790d56a6a.png)